cmd/tailscaled/tailscaled_windows.go

```diff
@@ -63,6 +66,7 @@ type ipnService struct {
 func (service *ipnService) Execute(args []string, r <-chan svc.ChangeRequest, changes chan<- svc.Status) (bool, uint32) {
 	changes <- svc.Status{State: svc.StartPending}
 
+	flushDNSOnSessionUnlock = winutil.GetRegInteger("FlushDNSOnSessionUnlock", 0) != 0
```

Comment:
I don't know what the Windows convention on initialisms is. Feel free to use FlushDnsOnSessionUnlock if that matches Windows style for registry keys. Unless we already picked one style in our other registry keys.

I was only commenting on the Go variable name before.

Reply:
Yeah, there isn't really a hard convention, so I might as well keep it mostly in sync with the Go variable.